examples/java/src/main/java/org/apache/ignite/example/table/KeyValueViewPojoExample.java

```diff
@@ -42,82 +39,75 @@ public static void main(String[] args) throws Exception {
-                            + "accountNumber INT PRIMARY KEY,"
-                            + "firstName     VARCHAR,"
-                            + "lastName      VARCHAR,"
-                            + "balance       DOUBLE)"
-            );
-        }
-
-        //--------------------------------------------------------------------------------------
-        //
-        // Creating a client to connect to the cluster.
-        //
-        //--------------------------------------------------------------------------------------
-
-        System.out.println("\nConnecting to server...");
-
         try (IgniteClient client = IgniteClient.builder()
                 .addresses("127.0.0.1:10800")
                 .build()
         ) {
-            //--------------------------------------------------------------------------------------
-            //
-            // Creating a key-value view for the 'accounts' table.
-            //
-            //--------------------------------------------------------------------------------------
-
-            KeyValueView<AccountKey, Account> kvView = client.tables()
-                    .table("accounts")
-                    .keyValueView(AccountKey.class, Account.class);
-
-            //--------------------------------------------------------------------------------------
-            //
-            // Performing the 'put' operation.
-            //
-            //--------------------------------------------------------------------------------------
-
-            System.out.println("\nInserting a key-value pair into the 'accounts' table...");
-
-            AccountKey key = new AccountKey(123456);
-
-            Account value = new Account(
-                    "Val",
-                    "Kulichenko",
-                    100.00d
-            );
-
-            kvView.put(null, key, value);
-
-            //--------------------------------------------------------------------------------------
-            //
-            // Performing the 'get' operation.
-            //
-            //--------------------------------------------------------------------------------------
-
-            System.out.println("\nRetrieving a value using KeyValueView API...");
-
-            value = kvView.get(null, key);
-
-            System.out.println(
-                    "\nRetrieved value:\n"
-                        + "    Account Number: " + key.accountNumber + '\n'
-                        + "    Owner: " + value.firstName + " " + value.lastName + '\n'
-                        + "    Balance: $" + value.balance);
-        } finally {
-            System.out.println("\nDropping the table...");
-
-            try (
-                    Connection conn = DriverManager.getConnection("jdbc:ignite:thin://127.0.0.1:10800/");
-                    Statement stmt = conn.createStatement()
-            ) {
-                stmt.executeUpdate("DROP TABLE accounts");
+            try {
```

Comment:
Please do not change unrelated examples. This example should be reverted. If you want to overcome the broken compatibility issue (https://issues.apache.org/jira/browse/IGNITE-27280) then please use the latest Ignite 3 build from `main`.